stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt

```diff
@@ -91,12 +91,22 @@ public fun Channel.getLastMessagePreviewText(
                     )
                 }
 
-                val attachmentText = message.attachments
-                    .takeIf { it.isNotEmpty() }
-                    ?.mapNotNull { attachment ->
-                        attachment.title ?: attachment.name
-                    }
-                    ?.joinToString()
+                val attachmentText: String? = if (message.attachments.isNotEmpty()) {
```

Comment:
How about extracting the `message.attachments` as a local value? Because it's repeatedly used. 😃